build.sh

```diff
@@ -222,6 +224,8 @@ case "$TRAVIS_OS_NAME" in
     signcode -t http://timestamp.digicert.com -a sha1 -spc .travis/ob1.cert.spc -pvk .travis/ob1.pvk -n "OpenBazaar $PACKAGE_VERSION" dist/win64/OpenBazaar2-$PACKAGE_VERSION-Setup-64.exe
     signcode -t http://timestamp.digicert.com -a sha1 -spc .travis/ob1.cert.spc -pvk .travis/ob1.pvk -n "OpenBazaarClient $PACKAGE_VERSION" dist/win64/OpenBazaar2Client-$PACKAGE_VERSION-Setup-64.exe
 
+    mv dist/RELEASES dist/win32/RELEASES
```

Comment:
This is in the win64 section, should it be win64 instead of win32?

Reply:
No we need to wait until all the builds are done and then move the file.

Reply:
I tested the build process locally and it looked good when simulating the Travis build.